File: app/src/main/java/wxm/KeepAccount/ui/data/show/calendar/ACCalendarShow.java
package wxm.KeepAccount.ui.data.show.calendar;

import android.os.Bundle;

import wxm.androidutil.Switcher.ACSwitcherActivity;


/**
 * for calendar
 * Created by WangXM on 2016/12/4.
 */
public class ACCalendarShow extends ACSwitcherActivity<FrgCalendarShow> {
    @Override
    protected void setupFragment(Bundle bundle) {
        addFragment(new FrgCalendarShow());
    }
}

File: app/src/main/java/wxm/KeepAccount/ui/data/show/calendar/ACCalendarShow.kt
package wxm.KeepAccount.ui.data.show.calendar

import android.os.Bundle
import wxm.KeepAccount.ui.base.ACBase.ACBase

import wxm.androidutil.ui.activity.ACSwitcherActivity


/**
 * for calendar
 * Created by WangXM on 2016/12/4.
 */
class ACCalendarShow : ACBase<FrgCalendarHolder>() {
    override fun setupFragment(savedInstanceState: Bundle?) {
        addFragment(FrgCalendarHolder())
    }
}
